src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -120,8 +120,11 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     network = dataprocConfig.vpcNetwork.map(VPCNetworkName),
     targetTags = List(NetworkTag(dataprocConfig.networkTag)))
 
-  // Startup scripts to install on the cluster master node.
-  // This allows Jupyter and Welder to start back up after a cluster is resumed.
+  // Startup script to install on the cluster master node. This allows Jupyter to start back up after
+  // a cluster is resumed.
+  //
+  // The || clause is included because older clusters may not have the run-jupyter.sh script installed,
```

Comment:
I notice the comment `The || clause ...` is duplicated down below. Probably can remove it here.